example/main.go

```diff
@@ -108,7 +108,7 @@ func resolveSecretReference(client *onepassword.Client, vaultID, itemID, fieldID
 
 func createAndGetItem(client *onepassword.Client) onepassword.Item {
 	sectionID := "extraDetails"
```

Comment:
Should this section ID be at the beginning of this example?

Reply:
Yes, in Go, the section ID is actually a pointer to a string, to ensure that it can be referenced both within the section, but also within the item field's section ID.

So this needs to be defined prior to the instantiation of the item.